### PR TITLE
Screen/result

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added result screen

### CODE REVIEW

File: category.html [hunk 1]
```diff
@@ -17,9 +17,9 @@
...
     <main class="container h-full w-full col items-center">
         <section class="category-section w-full text-light">
-            <h1 class="text-3xl text-bold mb-4 text-center">Choose <span class="text-secondary">Trivia</span> Category
+            <h1 class="text-3xl text-bold mb-4 text-center">Choose <span class="text-primary">Trivia</span> Category
             </h1>
-            <div class="row flex-gap-1 justify-center">
+            <div class="row gap-1 justify-center">
                 <div class="card w-60 bg-primary">
                     <div class="card-header">
                         <img src="images/classical.png" alt="" class="img h-40 w-full cover">
```
Comment (codewithrishu) [inline category.html]: alt tag should not be empty

**Review by codewithrishu — APPROVED** [review]
looks good to merge

File: category.html (hunk 1 above)
Comment (sanjitsarkar) [reply category.html]: fixed

**Review by sanu29 — APPROVED** [review]
Approved

**Review by HeyNitin — APPROVED** [review]
LGTM

**Comment by HeyNitin** [comment]
You can resolve the conflicts and merge

File: css/style.css [hunk 2]
```diff
@@ -1,16 +1,362 @@
...
+    left: 0;
+    box-shadow: var(--bx-sh-primary-1);
+  }
+  .filter-btn {
+    display: block;
+  }
+}
+.show {
+  display: flex !important;
+}
```
Comment (darshandamre) [inline css/style.css]: avoid using `!important`

**Review by darshandamre — COMMENTED** [review]
one suggestion from my side and solve the merge conflicts

**Review by VLeads — APPROVED** [review]
I am unable to find the result page in UI. I think you have missed to link that. Please check it.

I have seen that on some places you have used "!important". It is suggested to not use that until no option left. You can handle that thing by specificity. If not then debug that what is causing the issue.

Rest all you have done nice work. Just take a look at the suggestions that I gave.
Code looks good.
And nice work on UI too !👍

File: css/style.css (hunk 2 above)
Comment (sanjitsarkar) [reply css/style.css]: I could not find a fix for this as this property gets override

**Pull request merged**